<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StackedWidget</class>
 <widget class="QWidget" name="StackedWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>250</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>71</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>toolbtn</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/resource/info.png</normaloff>:/resource/info.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>10</y>
         <width>97</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>CheckBox</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>40</y>
         <width>171</width>
         <height>114</height>
        </rect>
       </property>
       <property name="title">
        <string>GroupBox</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="radioButton">
          <property name="text">
           <string>RadioButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_2">
          <property name="text">
           <string>RadioButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_3">
          <property name="text">
           <string>RadioButton</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QCommandLinkButton" name="commandLinkButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>187</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>CommandLinkButton</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>85</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>519</width>
            <height>484</height>
           </rect>
          </property>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>50</y>
             <width>120</width>
             <height>80</height>
            </rect>
           </property>
           <property name="title">
            <string>GroupBox</string>
           </property>
          </widget>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>210</y>
             <width>120</width>
             <height>80</height>
            </rect>
           </property>
           <property name="title">
            <string>GroupBox</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_2">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>240</y>
             <width>99</width>
             <height>27</height>
            </rect>
           </property>
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>360</y>
             <width>120</width>
             <height>80</height>
            </rect>
           </property>
           <property name="title">
            <string>GroupBox</string>
           </property>
          </widget>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>480</y>
             <width>120</width>
             <height>80</height>
            </rect>
           </property>
           <property name="title">
            <string>GroupBox</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
